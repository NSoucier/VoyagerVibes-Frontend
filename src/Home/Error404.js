function Error404() {
    return (
        <h4 id="error404">Test, this page could not be found (Error 404).</h4>
    )
}

export default Error404;