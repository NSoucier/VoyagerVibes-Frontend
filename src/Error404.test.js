import React from "react";
import { render } from "@testing-library/react";
import Error404 from "./Error404";

// smoke test
it("renders without crashing", function() {
  render(<Error404 />);
});

// snapshot test
it("matches snapshot", function() {
    const {asFragment} = render(<Error404 />);
    expect(asFragment()).toMatchSnapshot();
});

// it("displays form to generate itinerary", () => {
//   const { container } = render(<Error404 />);

//   // console.log('mmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmmm', container)
//   expect(
//       container.querySelector('button')
//   ).toBeInTheDocument();
// });