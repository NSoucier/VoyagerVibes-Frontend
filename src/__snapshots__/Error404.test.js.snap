// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <h4
    id="error404"
  >
    Sorry, this page could not be found (Error 404).
  </h4>
</DocumentFragment>
`;
